README.md

```diff
@@ -42,4 +38,4 @@ The following property can be configured:
 | colorLight | Background color        | "#000"        |
 | imageSize  | Size of the image in px | 150           |
 | showRaw    | Show raw text           | true          |
-| text       | The text to be encoded  |               |
+| text       | The text to be encoded  | 'Your Text'   |
```

Comment:
This column is used for default value. There you not "Your text" as a default value.

Reply:
yes, I have added it because people could understand it more clearly if you want it as blank then it is up to you. But according to my understanding w should define some value so that other people found it easier to use it. ;)

Reply:
Well, this is a documentation which describes the default value for this part of the config hardcoded in the code. And there is no such value hardcoded. The "default" value is not an "example".

Reply:
okay, I got your point I will just leave it blank as it is. :)